src/functions/users.rs

```diff
@@ -66,6 +66,10 @@ pub fn new_user(username: &str, hasroot: bool, password: &str, do_hash_pass: boo
             files::append_file("/mnt/etc/sudoers", "\nDefaults pwfeedback\n"),
             "Add pwfeedback to sudoers",
         );
+        files_eval(
+            files::create_directory("/mnt/var/lib/AccountsService"),
```

Comment:
Shouldn't it create the path `/mnt/var/lib/AccountsService/users` ?

Reply:
yes it should